src/renderer/components/ft-video-player/ft-video-player.js

```diff
@@ -602,12 +604,39 @@ export default defineComponent({
         // https://github.com/videojs/video.js/blob/v7.13.3/docs/guides/components.md#default-component-tree
         this.player.controlBar.progressControl.seekBar.playProgressBar.removeChild('timeTooltip')
 
+        // Count the number of visible elements in progress-holder > mouse-display and
+        // update vertical offset of thumbnail accordingly
+        const seekBar = this.player.controlBar.progressControl.seekBar.el()
+        const mouseDisplay = seekBar.querySelector('.vjs-mouse-display')
+        seekBar.addEventListener('mousemove', () => {
+          const children = Array.from(mouseDisplay.children)
+          const visibleChildren = children.filter((child) => {
+            const style = window.getComputedStyle(child)
+            const visibility = style.getPropertyValue('visibility')
+            return visibility === 'visible'
+          })
+
+          if (visibleChildren.length === 0) {
+            return
+          }
```

Comment:
I am not sure if the blank line should be kept, up to you
```suggestion
          if (visibleChildren.length === 0) { return }
```